README.md

```diff
@@ -1,49 +1,105 @@
 
-1. Clone the repository.
+### Repository fork
 
-```bash
-$ git clone git@github.com:vuestorefront/storefront-nuxt3-boilerplate.git
-```
+We recommend working with a fork of this repository. A fork allows you to easily incorporate updates from this boilerplate into your own codebase.
 
-2. Install all dependencies (read here [How to install Yarn 3](https://yarnpkg.com/getting-started/install)).
+1. [Create a fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo).
+2. [Clone the forked repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository).
 
-```bash
-$ yarn install
-```
+### Configuration
+
+- Open [`middleware.config.ts`](./apps/server/middleware.config.ts) and replace `url` with the URL of your plenty system.
+- Open [`i18n.config.ts`](./apps/web/i18n.config.ts) and [`nuxt.config.ts`](./apps/web/nuxt.config.ts) to update the locales. The locale settings have to match the plenty system settings. The `fallbackLocale` and `defaultLocale` have to match the system default language.
+- Open [`nuxt.config.ts`](./apps/web/nuxt.config.ts) to manage your `pwa.workbox` settings. [Workbox](https://developer.chrome.com/docs/workbox/) provides libraries and tools for Progressive Web App functionality.
+- Open [`tailwind.config.ts`](./apps/web/tailwind.config.ts) to update your theme.
+- Open [`cookie.config.ts`](./apps/web/cookie.config.ts) to manage your consent cookies.
+
+### Starting the app
 
-3. Run development server. App will be served with hot reload at [localhost:3000](http://localhost:3000/).
+1. Install all dependencies.
+2. Start the development server. The app will be served with hot reload at [localhost:3000](http://localhost:3000/).
 
 ```bash
+$ yarn
 $ yarn dev
 ```
 
-## Features:
+## Deployment
+
+This section describes how to deploy your app on your live domain.
+
+### plenty system
+
+1. Log into your plenty system.
+2. Go to **CMS » Deployment**.
+3. Click on :plus: to add a new application.
+4. Fill in the required information.
+5. **Create** the application.
+
+### GitHub
+
+1. In your forked repository, [create 2 encrypted secrets](https://docs.github.com/en/actions/security-guides/encrypted-secrets#creating-encrypted-secrets-for-a-repository):
+
+    - `URL_ENDPOINT`: TODO
+    - `URL_ENDPOINT_TOKEN`: The **Access key** from the application in your plenty system.
+
+2. In your forked repository, enable the GitHub Action `Add zip file to S3 bucket` from `upload.yml`. Update the trigger conditions to determine when to upload your repository to your plenty system.
+
+## Resources
+
+### Features
 
 - [Turborepo](https://turbo.build/) remote cache build system, with blazingly fast execution of commands (build, lint, test etc.) on your local machine.
```

Comment:
What is this about?

Reply:
It's one of the technologies used. I haven't looked into the details myself.